### ISSUE TITLE
Default Parameters For onMinimize, onMaximize, etc.

### ISSUE THREAD

**Issue by yikuansun** [AlexTorresSk/custom-electron-titlebar#174]
**Describe the bug**
I'm a little confused about why this is an issue, considering that older releases of custom-electron-titlebar did not require developers to write out the code to make the titlebar buttons work in the normal, standard way. However, in v4.0.1, I can't leave parameters like onMinimize and onMaximize blank without errors.
I'm not too familiar with TypeScript, but the parameters should default to the following:
```
    onMinimize: function() { BrowserWindow.getFocusedWindow().minimize(); },
    onMaximize: function() { BrowserWindow.getFocusedWindow().isMaximized()?BrowserWindow.getFocusedWindow().unmaximize():BrowserWindow.getFocusedWindow().maximize(); },
    onClose: function() { BrowserWindow.getFocusedWindow().close() },
    isMaximized: function() { return BrowserWindow.getFocusedWindow().isMaximized() },
    onMenuItemClick: function() { console.log("nothing"); }
```

**To Reproduce**
```
var titlebar = new customTitlebar.Titlebar({
	backgroundColor: customTitlebar.Color.fromHex("#474747"),
    menu: null
}); // used to work - no longer does
```

**Expected behavior**
custom-electron-titlebar will know that since onMinimize, onMaximize, etc. were left blank, it should make the default buttons.

**Desktop (please complete the following information):**
 - OS: Windows
 - Electron version 11.1.1
 - Node Version 14.16.0

**Comment by Araxeus** [comment]
How about moving the whole titlebar constructor to the Main process and do those events internally?

right now custom-electron-titlebar only works in the renderer process right? thats what causing the problems

**Comment by yikuansun** [comment]
Thanks @AlexTorresSk for the clarification, and thanks @Araxeus for the proposed solution.
I didn't realize that'd it'd be possible, but using the constructor from the main process is definitely a lot cleaner and more robust.

**Comment by Araxeus** [comment]
Well dunno why this wasn't closed automatically... the linked pull request was merged so pretty weird

anyways I fixed this issue in #175

(can now just require a main script in main process **once** on load + another one to attach to BrowserWindow - see example in  https://github.com/Araxeus/custom-electron-titlebar/tree/v4.1.0/example)
